docs/source/release-notes.rst

```diff
@@ -1,6 +1,20 @@
 Release Notes
 =============
 
+Version 0.4.0
+_____________
+- Streams: 
+    - Added support for multiple `queues` in `read_stream` and `write_stream` configuration, allowing to
+produce and consumer events in parallel from different sources. Engine automatically manages
+independent streams for each queue and ensures a message read from a queue are propagated
+dowstream using same queue.
+- Web: 
+    - Support for custom response `content-type` in `PostProcessHook`, i.e. to return `text/plan` or
```

Comment:
text/plain

Reply:
Fixed in https://github.com/hopeit-git/hopeit.engine/pull/70/commits/27e6822832a6535a4bf5a4e22629cbceab3994ad